client/devdocs/design/typography.jsx

```diff
@@ -43,10 +46,28 @@ export default class Typography extends React.PureComponent {
 						</p>
 					</Card>
 
+					<h3>How to use:</h3>
+
+					<p>
+						The <code>$sans</code> Sass variable will output the sans-serif font stack.
+					</p>
+
+					<pre>
+						<code>
+							.design__typography-sans &#123;
+							{ '\n\t' }font-family: $sans;
+							{ '\n\t' }font-size: 16px;
```

Comment:
I suggest you make this 15px, since it is our default size.

Reply:
Even though it's pretty silly that we don't use our own typo scale :).

Reply:
I was wondering about that, because of the typographic scale, but also because we seem to use 16px in various places around Calypso already (I did a very basic search in GitHub for `font-size` and we're allllll over the place). Should we take this opportunity to say it's 16px, and make an effort to bump the font size up when we're using 15px elsewhere?

Reply:
16px seems to be popular. Maybe that can be an on going side project.